pkg/scheduler/scheduler.go

```diff
@@ -36,9 +36,11 @@ type Scheduler struct {
 	cache          schedcache.Cache
 	config         *rest.Config
 	actions        []framework.Action
+	actionOptions  map[string]map[string]string
```

Comment:
this options should be included in Action instead of scheduler.

Reply:
this is not used any more. removed.